mlf_core/create/templates/mlflow/mlflow_pytorch/{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug_no_hyphen }}/model/model.py

```diff
@@ -1,27 +1,55 @@
+from torch.autograd import Variable
+import pytorch_lightning as pl
 import torch
-import torch.nn as nn
-import torch.nn.functional as F
+from argparse import ArgumentParser
+from pytorch_lightning.metrics.functional import accuracy
+from torch.nn import functional as F
 
 
-def create_model():
-    return Net()
+class LightningMNISTClassifier(pl.LightningModule):
+    def __init__(self, len_test_set: int, **kwargs):
+        """
+        Initializes the network
+        """
+        super(LightningMNISTClassifier, self).__init__()
 
+        # mnist images are (1, 28, 28) (channels, width, height)
+        self.optimizer = None
+        self.conv1 = torch.nn.Conv2d(1, 32, 3, 1)
+        self.conv2 = torch.nn.Conv2d(32, 64, 3, 1)
+        self.dropout1 = torch.nn.Dropout2d(0.25)
+        self.fc1 = torch.nn.Linear(9216, 128)
+        self.dropout2 = torch.nn.Dropout2d(0.25)
+        self.fc2 = torch.nn.Linear(128, 10)
+        self.args = kwargs
+        self.len_test_set = len_test_set
 
-def create_parallel_model():
-    return DataParallelPassthrough(Net())
+    @staticmethod
+    def add_model_specific_args(parent_parser):
+        parser = ArgumentParser(parents=[parent_parser], add_help=False)
 
+        parser.add_argument(
+            "--num_workers",
+            type=int,
+            default=3,
+            metavar="N",
+            help="number of workers (default: 3)",
+        )
+        parser.add_argument(
```

Comment:
dunno about that formatting?

Reply:
Y i think i will reformat the workers option